qiskit/_util.py

```diff
@@ -97,3 +105,57 @@ def _has_connection(hostname, port):
         return True
     except Exception:  # pylint: disable=broad-except
         return False
+
+
+def about():
+    """ Returns information about the Qiskit installation.
+    """
+    hardware_info = local_hardware_info()
+    n_qubits = int(log2(local_hardware_info()['memory']*(1024**3)/16))
+
+    print("")
+    print("Qiskit Details")
+    print("==============")
+    print("Package         Version Number")
+    print("------------------------------")
+    print("Qiskit-terra:       %s" % qiskit.__version__)
+    print("Numpy:              %s" % numpy.__version__)
+    print("Scipy:              %s" % scipy.__version__)
+    print("Networkx:           %s" % nx.__version__)
```

Comment:
rather than hardcoding these, why not get them from requirements.txt?
because say we remove networkx in the future, someone then has to remember to update this.

Also, do the versions of dependencies make a difference to the user experience (assuming they meet the minimum requirement)?

Reply:
These are not hardcoded but grabbed from the modules.  These numbers are likely not the minimum values given in the requirements.txt.  

If people remove packages, then of course you need to update stuff.  That would be a one-line change.  If I include a test, then in your situation, the test would fail telling people to remove the code in question.

So they could.  What it does help is with bug reporting.  It could be that new versions cause trouble, or version mismatches, and this info can be included in a bug report.  Basically the "Information" section that the top of our bug reports could be replace with just the return from this function.

Reply:
Oh I see what your saying.  Because I do not want to list all the requirements.  `psutil`, `ply` are probably not as important as the more 'core' packages above.